test/extensions/quic_listeners/quiche/integration/quic_http_integration_test.cc

```diff
@@ -391,9 +393,13 @@ TEST_P(QuicHttpIntegrationTest, TestDelayedConnectionTeardownTimeoutTrigger) {
             1);
 }
 
+// Only run on platforms that support BPF
+#if defined(SO_ATTACH_REUSEPORT_CBPF) && defined(__linux__)
```

Comment:
Why ifdef this out? Doesn't it pass on non-bpf platforms? (I'm basing that assumption on the comment below that the runtime override is a no-op on non-bpf platforms).

Reply:
It does pass, but may give the wrong impress on non-BPF platforms for someone taking a quick glance that BPF support somehow works, ifdef-ing it out is mainly a clarification

Also, we are having issues with this test timing out, not running the various parameterized runs of this test limits test runtime a bit

Reply:
Ah, I think the test really should be renamed, and always run. The name should probably be "Default mode" or something. It is testing that, given no extra config, quic works on this platform, which is why I think it should run on all platforms.

It is doing the same thing as the next test on non-bpf platforms, but the goal is to ensure that both configurations work on all platforms.